api/openapi.yml

```
@@ -15,6 +15,12 @@ paths:
     get:
       summary: Get documents
       description: Get a list of documents that are either public or associated with the current user
+      parameters:
+        - in: query
+          name: hasParent
```

```suggestion
          name: has_parent
```
https://api.gov.au/standards/national_api_standards/naming-conventions.html#message-format

---

Fixed